backend/core/src/user-preferences/entities/user-preferences.entity.ts

```diff
@@ -21,7 +21,10 @@ export class UserPreferences {
   @Expose()
   sendSmsNotifications?: boolean
 
-  @ManyToMany(() => Listing, (listing) => listing.favoritedPreferences, { nullable: true })
+  @ManyToMany(() => Listing, (listing) => listing.favoritedPreferences, {
+    nullable: false,
+    cascade: true,
```

Comment:
Prolly can remove this until we figure out what the issue is with the backend calls.

Reply:
updated